third_party/opa/README.md

```diff
@@ -0,0 +1,72 @@
+# Example
+
+### Add a policy
+
+```bash
+kubectl create configmap wkcpolicy.rego --from-file=wkcpolicy.rego -n irltest3 -o yaml --dry-run=client | kubectl label -f- --dry-run=client -o yaml --local openpolicyagent.org/policy=rego | kubectl apply -f -
+
+kubectl create configmap wkcpolicy.rego  --from-file=wkcpolicy.rego --from-file=data_policies/action_struct.rego --from-file=data_policies/auditing_struct.rego --from-file=data_policies/helper_functions.rego --from-file=data_policies/input_reader.rego --from-file=data_policies/taxonomies_unification.rego --from-file=data_policies/verify_correct_input.rego --from-file=data_policies/medical_taxonomies.json -n irltest3 -o yaml --dry-run=client | kubectl label -f- --dry-run=client -o yaml --local openpolicyagent.org/policy=rego | kubectl apply -f -
+
+kubectl create configmap wkcpolicy.rego  --from-file=main=wkcpolicy.rego --from-file=main=data_policies/action_struct.rego --from-file=main=data_policies/auditing_struct.rego --from-file=main=data_policies/helper_functions.rego --from-file=main=data_policies/input_reader.rego --from-file=main=data_policies/taxonomies_unification.rego --from-file=main=data_policies/verify_correct_input.rego --from-file=main=data_policies/medical_taxonomies.json -n irltest3 -o yaml --dry-run=client | kubectl label -f- --dry-run=client -o yaml --local openpolicyagent.org/policy=rego | kubectl apply -f -
+
+
+
+kubectl create configmap wkcpolicy.rego  --from-file=data_policies -n irltest3 -o yaml --dry-run=client | kubectl label -f- --dry-run=client -o yaml --local openpolicyagent.org/policy=rego  --local openpolicyagent.org/data=opa | kubectl apply -f -
+
+
+
+
+
+
+kubectl create configmap wkcpolicy.rego --from-file=wkcpolicy.rego -n irltest3 -o yaml --dry-run=client | kubectl label -f- --dry-run=client -o yaml --local openpolicyagent.org/policy=rego | kubectl apply -f -
```

Comment:
When storing all the policies etc in configmaps please keep in mind that there is a limit of 1MiB per configmap. So for larger policy definitions there may be problems.

Reply:
It's configmap per rego, as used in opa kube-mgmt.

Reply:
Generally see https://ibm.github.io/the-mesh-for-data/docs/usage/opa/update_policies/ for what we have today (before this PR)